site/app/templates/submission/EditTeamName.twig

```diff
@@ -5,15 +5,18 @@
     <p class="std-margin">
         Please fill in an appropriate name for your team if you would like.
     </p>
+    
     <div class="flex-col" id="team-name-modal">
         <input id="team-name-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
         <div class="edit_user_row">
             <label for="team-name-change">Team Name:&emsp;</label>
             <input type="text"
-                   name="team_name"
-                   value="{{ team_name }}"
-                   id="team-name-change"
-                   data-current-name="{{ team_name }}"/>
+                name="team_name"
+                value="{{ team_name }}"
+                id="team-name-change"
+                data-current-name="{{ team_name }}"
+                maxlength="{{ team_name_length }}"
```

Comment:
```suggestion
                max-length="{{ team_name_length }}"
```

To keep the variables consistent with others, there should be a hyphen.

Reply:
got it!

Reply:
@IDzyre , i just observed that maxlength is a pre defined html attribute at client side
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength 
So, its ok to revert it back from max-length right ?

Reply:
Yes

Reply:
Hey @IDzyre, reverted to maxlength and fixed those lint errors
You may run the tests, thanks.